package.json

```diff
@@ -208,7 +208,7 @@
   "private": true,
   "dependencies": {
     "@department-of-veterans-affairs/formation": "^6.2.2",
-    "@department-of-veterans-affairs/formation-react": "^4.4.2",
+    "@department-of-veterans-affairs/formation-react": "^4.4.3",
```

Comment:
This will likely have to change to `4.5.0`

Reply:
@erikphansen , I'll check in w/ you for final version-bump once companion PR94's approved.